modules/lldp/output.c

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include "gr_lldp.h"
+#include "lldp_priv.h"
+
+#include <gr_control_input.h>
+#include <gr_eth_input.h>
+#include <gr_eth_output.h>
+#include <gr_graph.h>
+#include <gr_ip4_control.h>
+#include <gr_log.h>
+#include <gr_mbuf.h>
+#include <gr_port.h>
+
+#include <rte_ether.h>
+#include <rte_graph_worker.h>
+#include <rte_malloc.h>
+#include <rte_version.h>
+
+enum {
+	OUTPUT = 0,
+	ERROR,
+	EDGE_COUNT,
+};
+
+extern struct gr_lldp_conf_common_data lldp_ctx;
+extern struct gr_lldp_conf_iface_data lldp_iface_ctx[RTE_MAX_ETHPORTS];
+
+const struct rte_ether_addr lldp_dst = {
+	.addr_bytes = {0x01, 0x80, 0xc2, 0x00, 0x00, 0x0e},
+};
```

Comment:
This is already defined in `control.c` as `lldp_dst2`. I'm OK with duplicating but at least add `static`.